src/components/EventCard.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Grid, Typography, useMediaQuery, Button } from '@mui/material';
+import { DateRange } from '@mui/icons-material';
+import LaunchIcon from '@mui/icons-material/Launch';
+import Image from 'next/image';
+import theme from '../theme';
+import { Event } from '../utils/types';
+
+type EventCardProps = {
+  data: {
+    events: Event[];
+  };
+};
+
+const EventCard = ({ data }: EventCardProps) => {
+    const event = data.events[0];
```

Comment:
you can pass it only one event, this card will be used to map over the whole events array, so just a single one. but you can update it later too 👍

Reply:
I'll update it when I work on events component.